packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -314,6 +321,8 @@ library LibProving {
                 // In scenarios where this transition is not the first one, we
                 // straightforwardly reset the transition prover to address
                 // zero.
+                // TODO(Brecht): Is it sure that in all cases all the neccessary data is stored
```

Comment:
I ran through different branches in the proving code and it seems all fileds are initialized.